demo/app.py

```diff
@@ -56,6 +56,7 @@ def gen_button_clicked():
     st.session_state[gen_button] = True
 
 
+sample_rate = 24000
```

Comment:
why is this now hardcoded here?

Reply:
its both hardcoded, and set in [line 170](https://github.com/mozilla-ai/document-to-podcast/blob/30c050bc3fffdb253aeedf6f6f05b5d2817fae6c/demo/app.py#L170). The issue is that now that we only define `speech_model` inside the `Generate Podcast` section, when the generation finishes, streamlit "refreshes", the `speech_model` is no longer defined anywhere and then it reaches `audio_np = stack_audio_segments` with an undefined `speech_model`. So somehow we need to set the `sample_rate` in `stack_audio_segments` without relying on `speech_model`.

This PR is a hacky fix. The proper solution is probably the same as in #57 ... We need to use fragments

Reply:
> So somehow we need to set the sample_rate in stack_audio_segments without relying on speech_model.

This kind of makes the hole speech_model abstraction useless for the app

Reply:
yeap, thats true... but is your suggestion to use `_load_kokoro_tts` instead of `load_tts_model`? 
The `speech_model` abstraction still makes sense for `cli`, or potentially other models, so I wouldn't get rid of it.

Reply:
>  but is your suggestion to use _load_kokoro_tts instead of load_tts_model?
 
 Not really 😅 my suggestion is the same as before, to fix the streamlit state (#71)

Reply:
Ah fair enough, we agree :sweat_smile: I am quite packed this week with the release of the STT blueprint, so I dont have time to look into this right now. Maybe next week? What should we do till then?
1. leave the bug and fix it with #71 
2. patch it with this hacky PR